lib/src/file_picker.dart

```diff
@@ -161,6 +161,9 @@ abstract class FilePicker extends PlatformInterface {
   ///
   /// On the web, this function will start a download for the file with [bytes] and [fileName].
   /// If the [bytes] or [fileName] are omitted, this will throw an [ArgumentError].
+  /// Note: [saveFile] will always return null on the web. This is because the web does not
```

Comment:
Whatever is done behind the scenes for the Blob url is an implementation detail and can be omitted.

```suggestion
  /// On the web, this function will start a download for the file with [bytes] and [fileName].
  /// If the [bytes] or [fileName] are omitted, this will throw an [ArgumentError].
  /// The returned path for the downloaded file will always be `null`.
```